dist-git-to-src-git/README.md

```diff
@@ -0,0 +1,78 @@
+# Converting dist-git to source-git
+
+This should be similar to [How to source-git?], except that it isn't :)
+
+The main difference is that [How to source-git?] starts with the upstream
+history and applies the patches from dist-git.
+
+In this case though, the starting point is dist-git itself, so a few things
+need to be done differently.
+
+Here are the steps:
+
+*Note:* things bellow were done on git.centos.org/rpms/rpm.
+
+1. Fetch the sources from the lookaside cache.
+
+   This will create some `tar.*` under `SOURCES`.
+
+2. Unpack the `tar` to `src/rpm`.
+
+3. Now applying the patches (from the `SOURCES` directory) can begin. In order
+   to decide which patches to apply, look in the spec-file. Use
+   [rebase-helper] for this.
+
+## Intermezzo: what should happen to all the patches?
+
+Here is what we could do with all the patches:
+
+- Patches which are used in the spec file and are applied to the sources will
+  be deleted, a.k.a they wont show up in the source-git tree.
+
+- Patches which are used in the spec file and cannot be applied - this is
+  where tooling should err out, as something bad happened in dist-git. Or? It
+  might also happen that the patch cannot be applied b/c it was applied
+  already? Will we able to tell this?
```

Comment:
such patches wouldn't be in the spec b/c the build would fail - if a patch cannot be applied, it's an error in the spec file

Reply:
True.